### PR TITLE
fix skybox render bug

### PR DESCRIPTION
**Pull request by godIsAProgrammer**
Re: #

### Changelog

*fix skybox render bug

-------

### Continuous Integration

This pull request:

* [ ] needs automatic test cases check.
  > Manual trigger with `@cocos-robot run test cases` afterward.
* [ ] does not change any runtime related code or build configuration
  > If any reviewer thinks the CI checks are needed, please uncheck this option, then close and reopen the issue.

-------

### Compatibility Check

This pull request:

* [ ] changes public API, and have ensured backward compatibility with [deprecated features](https://github.com/cocos/cocos-engine/blob/v3.5.0/docs/contribution/deprecated-features.md).
* [ ] affects platform compatibility, e.g. system version, browser version, platform sdk version, platform toolchain, language version, hardware compatibility etc.
* [ ] affects file structure of the build package or build configuration which requires user project upgrade.
* [ ] **introduces breaking changes**, please list all changes, affected features and the scope of violation.

### CODE REVIEW

File: editor/inspector/contributions/node.js [hunk 1]
```diff
@@ -1816,3 +1833,43 @@ exports.beforeClose = async function beforeClose() {
...
+ * @description 比较节点树，并渲染节点树
+ * @param oldTree 旧树
+ * @param newTree 新树
+ * @param resultTree 结果树
+ *
+ * */
+function diffRenderTree(oldTree, newTree, resultTree) {
+    // 获取较大树的 长度
+    let count = Math.max(oldTree.length, newTree.length);
+    console.log(oldTree, 'oldTree');
```
Comment (VisualSJ) [inline editor/inspector/contributions/node.js]: Do not upload debug code.

File: editor/inspector/contributions/node.js [hunk 2]
```diff
@@ -1816,3 +1833,43 @@ exports.beforeClose = async function beforeClose() {
...
+
+
+/**
+ * @description 比较节点树，并渲染节点树
+ * @param oldTree 旧树
+ * @param newTree 新树
+ * @param resultTree 结果树
+ *
+ * */
+function diffRenderTree(oldTree, newTree, resultTree) {
```
Comment (VisualSJ) [inline editor/inspector/contributions/node.js]: Why do we need DIFF? In most cases, there should be no difference

**Review by VisualSJ — APPROVED** [review]

**Review by VisualSJ — CHANGES_REQUESTED** [review]
Please answer some questions first

File: editor/inspector/contributions/node.js (hunk 2 above)
Comment (godIsAProgrammer) [reply editor/inspector/contributions/node.js:1845]: in new PR, I'm  not use DIFF, I will delete this function.

**Pull request closed without merging**